File: contractstudy/src/test/java/test/contractstudy/collectContracts/api/SpringAssertExtractor/java/testData/MultipleSpringAssert.java
package test.contractstudy.collectContracts.api.SpringAssertExtractor.java.testData;

import org.springframework.util.Assert;

import java.util.List;

public class MultipleSpringAssert {

  public void test1(List<String> list) {
    Assert.doesNotContain("contains no", "test");
    Assert.hasLength("contains no", "this has length");
    Assert.notNull(list);
  }

  public void test2(boolean flag, List<String> list) {
    Assert.isTrue(flag, "This is true");
    Assert.state(flag);
    Assert.notNull(list);
  }

  public void test3(List<String> list) {
    Assert.notEmpty(list);
  }
}

File: contractstudy/src/test/java/test/contractstudy/extractor/api/SpringAssertExtractor/kotlin/testData/MultipleSpringAssert.kt
package test.contractstudy.extractor.api.SpringAssertExtractor.kotlin.testData

import org.springframework.util.Assert

class MultipleSpringAssert {
    fun test1(list: List<String?>?) {
        Assert.doesNotContain("contains no", "test")
        Assert.hasLength("contains no", "this has length")
        Assert.notNull(list)
    }

    fun test2(flag: Boolean, list: List<String?>?) {
        Assert.isTrue(flag, "This is true")
        Assert.state(flag)
        Assert.notNull(list)
    }

    fun test3(list: List<String?>?) {
        Assert.notEmpty(list)
    }
}